vignettes/imputation.Rmd

```diff
@@ -23,7 +23,7 @@ missing, e.g., `"2019-10"` if the day and the time is unknown.
 
 The ADaM timing variables like `ADTM` (Analysis Datetime) or `ADY` (Analysis
 Relative Day) are numeric variables. They can be derived only if the date or
-datetime is complete. Therefore admiral provides imputation functions which fill
+datetime is complete. Therefore `{admiral}`provides imputation functions which fill
```

Comment:
There is a space missing after "`{admiral}`". The same applies to the next three changes.

Reply:
done